lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -218,29 +220,38 @@ export default class ScheduleExceptionForm extends Component<Props> {
             <ControlLabel>
               <small>On these dates*</small>
             </ControlLabel>
-            {ranges && ranges.map((range, index) => {
-              const startDateIndex = dates.findIndex(d => d === range.startDate)
-              const endDateIndex = dates.findIndex(d => d === range.endDate)
-              return (
-                <ExceptionDateRange
-                  index={index}
-                  key={uuidv4()}
-                  range={range}
-                  validationState={this._checkRangeValidation(startDateIndex, endDateIndex)}
-                  {...this.props}
-                />)
-            })}
-            {parsedDates && parsedDates.map((date, index) => {
-              const dateIndex = dates.findIndex(d => d === date)
-              return (
-                <ExceptionDate
-                  date={date}
-                  index={dateIndex}
-                  key={uuidv4()}
-                  validationState={this._checkValidation(`dates-${dateIndex}`)}
-                  {...this.props}
-                />)
-            })}
+            <FlipMove
+              enterAnimation='accordionVertical'
```

Comment:
Insert prop `key={activeEntity.id}` and comment about preventing animations when switching from one item to the next in the list of exceptions.